module_intros/environment.BondOrder.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<p>Is it worthwhile to mention method chaining here anymore? I feel like that gets introduced at the tutorial level, otherwise it's just something that users will pick up on, but isn't something we need to call out in random places. Thoughts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>